packages/sdk-js/src/GrowthBook.ts

```diff
@@ -1146,8 +1195,34 @@ export class GrowthBook<
     else console.log(msg, ctx);
   }
 
+  public getDeferredTrackingCalls() {
+    return btoa(JSON.stringify(this._deferredTrackingCalls));
+  }
+
+  public fireDeferredTrackingCalls(data: string) {
```

Comment:
Would not take any arguments if we have a setter that is responsible for setting `deferredTracking`